### PR TITLE
test: add alert e2e testing

### PR DESCRIPTION
**Pull request by johnhwhite**
[AB#2194030](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2194030)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/314/storybook/)

Component Storybooks:

- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/314/storybooks/indicators-storybook/)

File: apps/indicators-storybook/src/app/alert/alert.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (Blackbaud-PaulCrowder) [inline apps/indicators-storybook/src/app/alert/alert.component.spec.ts:1]: Can we delete this file and the `app.component.spec.ts` file to cut down on the number of files created with these tests? We could also probably ditch `app.component.scss` and remove that from the component decorator.

Comment (johnhwhite) [reply apps/indicators-storybook/src/app/alert/alert.component.spec.ts:1]: I've updated the generator to not create the boilerplate files, and I've removed them from the `indicators-storybook` project.

**Comment by johnhwhite** [comment]
Test the pr comment command

**Review by Blackbaud-PaulCrowder — APPROVED** [review]

**Pull request merged**